lib/groonga-query-log/command/run-regression-test.rb

```diff
@@ -70,7 +85,14 @@ def run(command_line)
         tester = Tester.new(old_groonga_server,
                             new_groonga_server,
                             tester_options)
-        tester.run
+        tester_result = tester.run
+
+        if notifier_options[:mail_notification]
+          notifier = EMailNotifier.new(tester_result, @start_time, notifier_options)
```

Comment:
Passing elapsed time instead of start time is better. If we pass elapsed time measurement is a responsibility of this class not `EMailNotifier`.